<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="kd_petugasLabel">
       <property name="text">
        <string>kd_petugas</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kd_petugasLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nama_petugasLabel">
       <property name="text">
        <string>nama_petugas</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nama_petugasLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="no_teleponLabel">
       <property name="text">
        <string>no_telepon</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="no_teleponLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="levelLabel">
       <property name="text">
        <string>level</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="levelComboBox">
       <item>
        <property name="text">
         <string>admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>operator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dokter</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>256</width>
      <height>111</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>kd_petugas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nama_petugas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>no_telepon</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>password</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>level</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ubah</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCari">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCetak">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Cetak</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
